src/views/SettingsPage.vue

```diff
@@ -159,6 +215,27 @@ export default {
             alert(error);
           });
       }
+    },
+    addNewUser() {
+      const vm = this;
+      var user = new AV.User();
```

Comment:
You can use `const` instead of `var` because we're not gonna reassign anything to the same variable.